### DISCUSSION TITLE
Gemini TV

### DISCUSSION THREAD

**Discussion by varunfri** [iptv-org/community#695] [Q&A]
**Channel Name: Gemini Tv
**Channel ID:** GeminiTV.in
**Country:** India
**Language:** telugu
**Website:**https://sunnetwork.in/tv-channel-details.aspx?Channelid=4&channelname=GEMINI%20TV&LanguageID=2&Type=2
**Stream URL (optional):** xxx
**Notes (optional):** xxx

**Comment by freearhey** [comment]
Closed in favor of https://github.com/iptv-org/iptv/issues/19718
